samples/third-party/webgl2-half-float-array-converter.js

```diff
@@ -0,0 +1,98 @@
+
+var HALFFLOAT = HALFFLOAT || {};
+
+(function(){
```

Comment:
Did any code in this file come from another library?  For example, do we need to update LICENSE.md with licenses for new third-party code?